common/src/main/java/com/blamejared/crafttweaker/natives/item/component/ExpandMapId.java

```diff
@@ -0,0 +1,18 @@
+package com.blamejared.crafttweaker.natives.item.component;
+
+import com.blamejared.crafttweaker.api.annotation.ZenRegister;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import com.blamejared.crafttweaker_annotations.annotations.NativeTypeRegistration;
+import net.minecraft.world.level.saveddata.maps.MapId;
+import org.openzen.zencode.java.ZenCodeType;
+
+@ZenRegister
+@Document("vanilla/api/item/component/MapId")
+@NativeTypeRegistration(value = MapId.class, zenCodeName = "crafttweaker.api.item.component.MapId")
+public class ExpandMapId {
```

Comment:
static expansion method to create one?

Reply:
Should we? We already support calling `withMapId` with an integer parameter, but realistically only vanilla should be creating MapIds from values it is tracking.

Reply:
I f eel for consistency we should have it